pdns/dnsdist.cc

```diff
@@ -927,18 +1012,18 @@ try
       cout<<"Syntax: dnsdist [-C,--config file] [-c,--client] [-d,--daemon]\n";
       cout<<"[-p,--pidfile file] [-e,--execute cmd] [-h,--help] [-l,--local addr]\n";
       cout<<"\n";
-      cout<<"-a,--acl netmask      Add this netmask to the ACL\n";
-      cout<<"-C,--config file      Load configuration from 'file'\n";
-      cout<<"-c,--client           Operate as a client, connect to dnsdist\n";
-      cout<<"-d,--daemon           Operate as a daemon\n";
-      cout<<"-e,--execute cmd      Connect to dnsdist and execute 'cmd'\n";
-      cout<<"-g,--gid gid          Change the process group ID after binding sockets\n";
-      cout<<"-h,--help             Display this helpful message\n";
-      cout<<"-l,--local address    Listen on this local address\n";
-      cout<<"--supervised          Don't open a console, I'm supervised\n";
-      cout<<"                        (use with e.g. systemd and daemontools)\n";
-      cout<<"-p,--pidfile file     Write a pidfile, works only with --daemon\n";
-      cout<<"-u,--uid uid          Change the process user ID after binding sockets\n";
+      cout<<"-a,--acl netmask            Add this netmask to the ACL\n";
+      cout<<"-C,--config file            Load configuration from 'file'\n";
+      cout<<"-c,--client                 Operate as a client, connect to dnsdist\n";
+      cout<<"-d,--daemon                 Operate as a daemon\n";
+      cout<<"-e,--execute cmd            Connect to dnsdist and execute 'cmd'\n";
+      cout<<"-g,--gid gid                Change the process group ID after binding sockets\n";
+      cout<<"-h,--help                   Display this helpful message\n";
+      cout<<"-l,--local address          Listen on this local address\n";
+      cout<<"-s,--supervised             Don't open a console, I'm supervised\n";
```

Comment:
`-s` isn't hooked up in the `getopt_long`-call on line 990 as I wanted people to consciously set this option.